modules/alloy/src/main/ts/ephox/alloy/dropdown/DropdownUtils.ts

```diff
@@ -245,7 +245,7 @@ const makeSandbox = (
       // TODO: Add aria-selected attribute
       attributes: {
         id: ariaControls.id,
-        role: 'listbox'
+        ...detail.role?.fold( Fun.constant({}), (role) => ({ role })) ?? {}
```

Comment:
Why do we have an optional field with an `Optional` value 🤦 

please either make the field mandatory or the value unwrapped.

If there's really no other way... you can actually spread undefined. It feels dirty though.

```suggestion
        ...detail.role?.map((role) => ({ role })).getOr({})
```